cpp-package/example/unittests/unit_test_mlp_csv.sh

```diff
@@ -0,0 +1,63 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# This file is a unit test for mlp_csv.cpp example in 'example' directory.
+# The file
+#    1. Downloads the MNIST data,
+#    2. Converts it into CSV format.
+#    3. Runs the mlp_csv example and ensures that the accuracy is more than expected.
+#
+
+#!/bin/bash
+
+set -e # exit on the first error
```

Comment:
I am wondering why chose shell script for this? this example as Cpp, Py and sh..may be just use python here, WDYT?

Reply:
I chose shell script because I found it easier to invoke python and cpp programs from shell and process the output using grep.

There was no other particular reason.  We can revisit later and may be tie unit tests for all examples in single framework later.

Reply:
Python and check_call is more maintainable.

Reply:
Sure @larroy . I will consider these suggestions when I will start implementing the test framework.